docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -693,29 +781,21 @@ def my_s3_sensor(context):
 
 ### Using resources in sensors
```

Comment:
I know this isn't your change, but it looks like this page now has two similar sections:

- [This section](https://dagster-2yc4y0pgy-elementl.vercel.app/concepts/partitions-schedules-sensors/sensors#using-resources-in-sensors-1)
- [Your new section](https://dagster-2yc4y0pgy-elementl.vercel.app/concepts/partitions-schedules-sensors/sensors#using-resources-in-sensors)

Can we make the heading something different/more specific to the example itself? I can see these two sections being potentially confusing for users